client/src/components/Atoms/Layout.tsx

```diff
@@ -1,21 +1,52 @@
-import styled from 'styled-components'
+import styled, { css } from 'styled-components'
 
-export const CenterRow = styled.div`
+interface BoxProps {
+  justify?: 'start' | 'center' | 'end' | 'space-between' | 'space-around'
+  align?: 'start' | 'center' | 'end' | 'stretch' | 'baseline'
```

Comment:
Might want to name these props to directly match the CSS properties, e.g. `justifyContent` and `alignItems`, to avoid ambiguity with the many other CSS properties that start with `justify-` and `align-`, e.g. `justify-self` and `align-self`

Reply:
Ok, will do